src/transaction/log_applier.c

```diff
@@ -7215,8 +7215,6 @@ la_get_copied_log_info (const char *database_name, const char *log_path, INT64 p
 
   if (page_num >= 0)
     {
-      printf ("\n *** Copied Log Page Info. *** \n");
```

Comment:
The order of headers is not defined too, but you didn't change it tough it will simplify the code because it will change the result, and users may use it already. Why can this be changed, and does this have to be changed?

Reply:
@eido5 
I also want to know the answer to this question